### PR TITLE
Suggestion for LE2 #334

### PR DESCRIPTION
**Pull request by jkostiuk**
I reorganized the activities beginning with 1.2.12 into a single activity with multiple parts.

For facilitators, I think this is helpful as it reinforces that these four tasks are part of one unified thread of inquiry. It now provides, explicitly, an opportunity for Facilitators to have students revisit the first matrix again after unpacking student reasoning in the first three tasks.

### CODE REVIEW

File: source/linear-algebra/source/01-LE/02.ptx [hunk 1]
```diff
@@ -644,64 +644,82 @@ in many applications we use it for in <xref ref="EV"/>.
...
+            <li> Add row 3 to row 2 (<m>R_2+R_3 \rightarrow R_2</m>)
+            </li>
+            <li> Add row 2 to row 3 (<m>R_3+R_2 \rightarrow R_3</m>)
+            </li>
+            <li> Swap row 1 to row 2 (<m>R_1 \leftrightarrow R_2</m>)
+            </li>
+            <li> Add -2 row 2 to row 1 (<m>R_1-2R_2 \rightarrow R_1</m>)
+            </li>
+        </ol>
+    </task>
```
Comment (StevenClontz) [inline source/linear-algebra/source/01-LE/02.ptx:722]: This appears to be accidentally identical to the first task.

Comment (StevenClontz) [reply source/linear-algebra/source/01-LE/02.ptx:722]: oh wait well maybe I should read your PR description. Then again, instructors won't read the PR description either - I don't see how this repetition helps just by reading the activity book, but it's late and I should look again with fresh eyes in the morning.

Comment (jkostiuk) [reply source/linear-algebra/source/01-LE/02.ptx:722]: @siwelwerd can you clarify what you're proposing. I did a few things and I'm not clear if we're rejecting all of them. 

First I grouped the activities into one multi-part activity (do we like this or not?)

Secondly, are you suggesting adding a separate activity in which we revisit the first activity and then include an instructor note?

Comment (siwelwerd) [reply source/linear-algebra/source/01-LE/02.ptx:722]: I'm saying to reject what Steven proposed, but add an instructor note as to why it is repeated, as some instructors might think it's an error

Comment (jkostiuk) [reply source/linear-algebra/source/01-LE/02.ptx:722]: working on the note now.

**Comment by siwelwerd** [comment]
@jkostiuk I updated the style guide with the proper code for an instructor note

**Comment by StevenClontz** [comment]
It's still not clear to me what we're guiding students to inquire into by repeating the first task. What is the change you expect the students to make before/after?

**Comment by jkostiuk** [comment]
In my teams experience with this activity, students answers are all over the place the first time they see the matrix. Some want the 1 in the top left, but a bunch of others choose other options because they make a lot of zeroes. 

In the next two activities, students actually use the marked pivots strategically to clear the positions above and below. 

So, by revisiting, I’m hoping students might actually use the pivot as part of the strategy when they weren’t usually doing so before. 

Also, as an aside, I don’t actually expect students to recognize that it’s the same matrix.

**Comment by siwelwerd** [comment]
Maybe expanding the instructor note, and moving it to before the activity, would help.  With the callback, it's probably okay to leave the first task a little unresolved, and then resolve it with part (d).  But it would be helpful to write that out for instructors.

**Comment by jkostiuk** [comment]
took a stab at an expanded note @siwelwerd

**Comment by siwelwerd** [comment]
I think that's good!  @StevenClontz, is that more clear to you what is happening now?  Jordan and I talked about this previously so my view is colored by those conversations.

**Comment by StevenClontz** [comment]
Okay, with the expanded note, I now get the rationale for the revisit.

I think I have an idea for an alternative here we might consider, but that should not block this strict improvement.

**Review by StevenClontz — APPROVED** [review]

**Pull request merged**